tests/resources/telescope_positions-South-4MST.ecsv

```diff
@@ -46,8 +46,8 @@
 #       unit: *id001
 #       value: 3.0
 # schema: astropy-2.0
-telescope_name pos_x pos_y pos_z
-MST-01 70.0 70.0 10.0
-MST-02 -70.0 -70.0 100.0
-MST-03 70.0 -70.0 0.0
-MST-04 -70.0 -70.0 42.0
+telescope_name      pos_x      pos_y      pos_z
+MST-01               70.0       70.0       10.0
+MST-02              -70.0      -70.0      100.0
+MST-03               70.0      -70.0        0.0
+MST-04              -70.0      -70.0       42.0
```

Comment:
This telescope is sitting on top of the second one. I think that's a typo and either the `x` coordinates should be negative.